### PR TITLE
feat(STONEINTG-536): add 2 panels for SLO 6

### PR DESCRIPTION
**Pull request by jencull**
SEB created to ready timeseries panel
SEB created to ready violations panel

## Maintainers will complete the following section

- [x] Commit messages are descriptive enough ([hints](https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/))
- [x] Code coverage from testing does not decrease and new code is covered ([check the PR coverage on codecov](https://app.codecov.io/gh/redhat-appstudio/integration-service/pulls))
- [x] [Controllers diagrams](https://github.com/redhat-appstudio/integration-service/tree/main/docs) are updated when PR changes controllers code  (if applicable)

### CODE REVIEW

**Comment by MartinBasti** [comment]
I'm again asking for a snapshot

**Review by MartinBasti — APPROVED** [review]

**Comment by jencull** [comment]
> I'm again asking for a screenshot

image

**Comment by jencull** [comment]
> I'm again asking for a screenshot

image

**Comment by jencull** [comment]
> I'm again asking for a screenshot

image

**Review by dirgim — APPROVED** [review]
LGTM

**Comment by jencull** [comment]
> I'm again asking for a screenshot

image

File: config/grafana/dashboards/integration-service-dashboard.json [hunk 1]
```diff
@@ -616,7 +616,7 @@
         {
           "editorMode": "code",
           "exemplar": true,
-          "expr": "sum(rate(release_latency_seconds_bucket{le=10}[$__rate_interval])) by (job)/sum(rate(release_latency_seconds_bucket[$__rate_interval])) by (job)*100",
```
Comment (MartinBasti) [inline config/grafana/dashboards/integration-service-dashboard.json:619]: Why is this change in this PR?

Comment (jencull) [reply config/grafana/dashboards/integration-service-dashboard.json]: I had a merge conflict from a PR I merged earlier today, despite doing a pull. I have it fixed up now. I'll keep an eye to  make sure it all goes through ok.

**Review by MartinBasti — APPROVED** [review]

**Pull request merged**